jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/test/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPopupTest.java

```diff
@@ -58,6 +57,8 @@ public void errorGenerationTest() {
                                    errors,
                                    null);
 
-        verify(errorsOccurredList).setHTML(SafeHtmlUtils.fromTrustedString("<strong>" + error.getMessage() + "</strong><br/>" + error.getStacktrace() + "<br><br>"));
+        verify(errorsOccurredList).setText("errorMessage\n" +
+                                                   "stackTrace\n" +
+                                                   "\n");
```

Comment:
Just a cosmetic thing here. This String breakdown looks a bit off. Maybe fixing the indention so each part is exactly below the previous one would make it look nicer :)